lib/core/enhanceError.js

```diff
@@ -34,7 +34,7 @@ module.exports = function enhanceError(error, config, code, request, response) {
       columnNumber: this.columnNumber,
       stack: this.stack,
       // Axios
-      config: this.config,
+      config:this.config,
```

Comment:
format: style mismatch:
```suggestion
      config: this.config, code: this.code
```

`d426da53` ␣ should be inserted at column 14.
`7c330c10` Replace ⏎ with ␣ at column 26.